### DISCUSSION TITLE
Can I have data from two queries displayed on the same chart?

### DISCUSSION THREAD

**Discussion by dgaloray** [grafana/grafana-infinity-datasource#333] [Q&A]
I want the first query to display individual data points.  The second query returns the monthly average all the data points.  Both queries return timeseries data.  So the chart needs to look like a point scatter chart with a monthly avg. line chart over the top.

**Comment by yesoreyeram** [comment]
Look like a grafana question. Doesn't seems to plugin related.
Yes it is possible in grafana.
